bgpd/bgp_evpn.c

```diff
@@ -3494,6 +3533,17 @@ void bgp_evpn_handle_router_id_update(struct bgp *bgp, int withdraw)
 	}
 }
 
+/*
+ * Handle change to auto-RT algorithm - update and advertise local routes.
+ */
+void bgp_evpn_handle_autort_change(struct bgp *bgp)
+{
+	hash_iterate(bgp->vnihash,
```

Comment:
We should also change the import/export RTs for l3-vni to follow this encoding as well.

Reply:
I am not familiar with the concept of a L3 VNI. From my understanding, just having a non-zero IP address in type 2 routes is not enough, right? It seems there is an additiona VNI number. Could you point me to an RFC section?

Reply:
I think we can open a new bug/issue here and handle it at a separate commit. But we do need to follow the same encoding for l2-vnis and l3-vnis.